src/styles/notify/notification-hub.scss

```diff
@@ -24,6 +24,11 @@
     transform: translateX(-100%);
   }
 
+  // reset child icons color since by default the icon is lightened when the mouse is in hover state
+  #wpadminbar &:hover #wp-notify-hub .ab-icon:before {
```

Comment:
These hover states should probably apply to `:focus` as well, since keyboard nav should have the same experience